fronts-client/src/components/FrontsEdit/FrontContent.tsx

```diff
@@ -94,6 +94,113 @@ interface FrontState {
 	isCollectionsStale: boolean | undefined;
 }
 
+function getNextGroupTarget(
+	currentGroupId: string,
+	groupIds?: string[],
+	groupsData?: Group[],
+) {
+	if (!groupIds || !groupsData) {
+		return;
+	}
+	const currentIndex = groupIds.findIndex((id) => id === currentGroupId);
+	const nextGroupId = groupIds[currentIndex + 1];
```

Comment:
`groupsIds` could be an empty array, in which case this might cause a runtime error?